frontend/src/metabase/dashboard/hooks/use-location-sync.ts

```diff
@@ -4,36 +4,36 @@ import { replace } from "react-router-redux";
 import { usePrevious } from "react-use";
 import { omit } from "underscore";
 
-import { DEFAULT_EMBED_DISPLAY_OPTIONS } from "metabase/dashboard/constants";
+import { DEFAULT_DASHBOARD_DISPLAY_OPTIONS } from "metabase/dashboard/constants";
 import { parseHashOptions, stringifyHashOptions } from "metabase/lib/browser";
 import { useDispatch } from "metabase/lib/redux";
 import { isNullOrUndefined } from "metabase/lib/types";
 
 type SYNCED_KEY = "refresh" | "fullscreen" | "theme";
 
-const DEFAULT_DASHBOARD_EMBED_DISPLAY_OPTIONS = {
-  theme: DEFAULT_EMBED_DISPLAY_OPTIONS.theme,
+const DEFAULT_SYNCED_DASHBOARD_OPTIONS = {
```

Comment:
I want to avoid the term _display options_ here because we have the type for `display options` in some places e.g.
https://github.com/metabase/metabase/blob/563c0a5d16bb2913a64ed669cd591e3ae8558f9b/frontend/src/metabase/public/lib/types.ts#L28-L34

This is not quite _display options_, but _display params_

https://github.com/metabase/metabase/blob/563c0a5d16bb2913a64ed669cd591e3ae8558f9b/frontend/src/metabase/dashboard/types/embed-display-options.ts#L32-L40

as you can see these options modify the appearance of the resource (questions or dashboards) however, `fullscreen` and `refresh` here don't fall into this category, so I think the more plain _dashboard options_ would suit it better.